Sources/TSCUtility/SQLite.swift

```diff
@@ -15,6 +15,100 @@ import TSCBasic
 
 /// A minimal SQLite wrapper.
 public struct SQLite {
+    /// The location of the database.
+    public let location: Location
+
+    /// The configuration for the database.
+    public let configuration: Configuration
+
+    /// Pointer to the database.
+    let db: OpaquePointer
+
+    /// Create or open the database at the given path.
+    ///
+    /// The database is opened in serialized mode.
+    public init(location: Location, configuration: Configuration? = nil) throws {
+        self.location = location
+        self.configuration = configuration ?? Configuration()
+
+        var db: OpaquePointer?
+        try Self.checkError("unable to open database at \(self.location)") {
+            sqlite3_open_v2(
+                location.pathString,
+                &db,
+                SQLITE_OPEN_READWRITE | SQLITE_OPEN_CREATE | SQLITE_OPEN_FULLMUTEX,
+                nil
+            )
+        }
+
+        self.db = db!
```

Comment:
this code has no changed, I believe it should be safe

Reply:
`sqlite3_open_v2()` is documented to return a database handle if the call succeeds, and the error code is checked, so this should be safe.  In general I think force-unwrap is to be avoided when at all possible.  This code could throw `db` is nil even after succeeding, but since the code is unchanged this shouldn't be a problem in actual use.  It would be just to avoid the force-unwrap out of principle.

Reply:
addressed